service/mapper/mapper.go

```diff
@@ -102,14 +102,14 @@ func (m *Mapper) Stop(ctx context.Context) error {
 // run function; that would make it quite easy to resume from an arbitrary
 // point in the LedgerWAL and get rid of the related struct fields.
 
-func (m *Mapper) Run() error {
+func (m *Mapper) Run() (*trie.MTrie, error) {
```

Comment:
What's the point of this change?

Reply:
We need to be able to obtain the tree at the end of the run. Otherwise, how can we have the final tree when it's an internal variable of the mapper? I will change this to use a callback, though. That's a bit cleaner.